dist/api_data.py

```diff
@@ -721,6 +721,28 @@ def __ge__(self, other):
             this will update the value if one is already set''',
             min='1MB', max='10TB')
         ]),
+    Config('tiered_manager', '', r'''
+        tiered storage manager configuration options''',
+        type='category', subconfig=[
+            Config('object_target_size', '10M', r'''
```

Comment:
Is this literally the size of objects that we want to create on S3 (or equivalent)?  If so does it need to be part of the tiered configuration or could it be an attribute of the bottom tier (i.e., an option to log_structured allocation)?

Reply:
Good point. It needs to be somewhere else, and/or repeated as an option to the `WT_SESSION::tier_storage` API.